tests/fixtures.py

```diff
@@ -366,8 +359,10 @@ def policy_rate():
 
 
 @pytest.fixture(scope='module')
-def policy_value(application_economics, policy_rate):
-    value = policy_rate * application_economics.min_operator_seconds
+def policy_value(policy_rate):
+    # TODO constant?
+    min_operator_seconds = 60 * 60 * 24  # one day in seconds
```

Comment:
Good thought, in lieu of a full-blown class it will still be useful to ensure the same value is used during deployment either by fixturization, or use of a constant.

Reply:
✔️